### ISSUE TITLE
Spread operator not supported

### ISSUE THREAD

**Issue by natezerk** [vuedoc/md#5]
I have a spread operator in my vue component (...) and the generation failes with the error:

'SyntaxError: Unexpected token ...'

Seems it isn't supporting the spread operator.

**Comment by demsking** [comment]
Could you give me an example of your use of spread operator please?

**Comment by natezerk** [comment]
Example that is breaking:

```
      /**
      * Fires when the card is changed.
      */
      this.$emit('change', {
        bankAccount: { ...this.bankAccount },
        valid: !this.$v.$invalid
      })
```

**Comment by demsking** [comment]
Hum... Seems this is not yet supported by the `vuedoc/parser`.
I created the _[vuedoc/parser#7](https://github.com/vuedoc/parser/issues/7)_ to link with the current issue.

**Comment by natezerk** [comment]
Thanks alot @demsking

**Comment by demsking** [comment]
Fixed with [@vuedoc/md@1.0.1](https://github.com/vuedoc/md/releases/tag/v1.0.1)

**Comment by demsking** [comment]
**This project has been moved to https://gitlab.com/vuedoc/md**
